### PR TITLE
Correct env var name in configure-helper

### PR DESCRIPTION
**Pull request by mtaufen**
The variable should be KUBEPROXY_TEST_ARGS, not KUBE_PROXY_TEST_ARGS.

```release-note
Correct env var name in configure-helper
```

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/33848)

### CODE REVIEW

**Comment by wojtek-t** [comment]
lgtm

File: cluster/gce/gci/configure-helper.sh [hunk 1]
```diff
@@ -1131,7 +1131,7 @@ function start-rescheduler {
 function setup-kubelet-dir {
     echo "Making /var/lib/kubelet executable for kubelet"
     mount --bind /var/lib/kubelet /var/lib/kubelet/
```
Comment (vishh) [inline cluster/gce/gci/configure-helper.sh]: While you are there, can you change `--bind` to `-B` to be consistent?

**Review by vishh — APPROVED** [review]
Just one comment, otherwise LGTM. Feel free to self -lgtm this patch once tests are green.

File: cluster/gce/gci/configure-helper.sh (hunk 1 above)
Comment (mtaufen) [inline cluster/gce/gci/configure-helper.sh]: Done

**Comment by adityakali** [comment]
Thanks for the quick fix! Please also cherry-pick this for 1.4.1.

**Comment by mtaufen** [comment]
@k8s-bot unit test this issue #33838

**Comment by mtaufen** [comment]
@k8s-bot gci gke test this issue #IGNORE

**Comment by gmarek** [comment]
@mtaufen - thanks for fixing this!

**Comment by mtaufen** [comment]
@jessfraz @adityakali This still needs to be cherry-picked to 1.4.

**Comment by jessfraz** [comment]
cherry-pick opened in https://github.com/kubernetes/kubernetes/pull/37140

**Pull request merged**